public/app/app.ts

```diff
@@ -93,6 +95,7 @@ export class GrafanaApp {
       setLocale(config.bootData.user.locale);
       setWeekStart(config.bootData.user.weekStart);
       setPanelRenderer(PanelRenderer);
+      setPanelDataErrorView(PanelDataErrorView);
```

Comment:
@torkelo why do we need to set the error view via runtime? Can't it be a component used the same way as you do down below in the TimeSeriesPanel? I can't see the getter function for the PanelDataErrorView used anywhere in this PR.

Reply:
@dprokop not sure what you mean? How can it be used from plugins that way? 

There is no getter function, we export the PanelDataErrorView component from runtime (but implementation is in core). This is using the same way as we did for PanelRenderer

Reply:
My bad, haven't noticed the PanelDataErrorView eport from runtime. I see now, we need it to live in core for it to be able to interact with redux etc. Forget the comment above :)